x/foundation/genesis.go

```diff
@@ -111,6 +111,10 @@ func ValidateGenesis(data GenesisState) error {
 		return err
 	}
 
+	if data.GovMintLeftCount > 1 {
+		return sdkerrors.ErrInvalidType.Wrap("invalid govMintLeftCount(0 or 1)")
+	}
+
```

Comment:
The `1` should be replaced by a config, or at least by a constant.
Then, you may correct the corresponding error messages.
Also, I think `ErrInvalidType` here is inappropriate.

Reply:
I created x/fountdation/errors.go and added appropriate error.